CONTRIBUTING.md

```diff
@@ -652,6 +650,20 @@ rather than via `tox`. If you have installed the development packages in your py
 `pip install -r requirements-dev.txt`, then `ruff` and `black` will be available and can be run from
 the command line. See [`tox.ini`](tox.ini) for how `tox` invokes them.
 
+### Rust style and lint
+
+If you're working on Rust code the Rust formatting and lint checking are done
+using different tooling. Qiskit uses [rustfmt](https://github.com/rust-lang/rustfmt) for
+code formatting, and you can simply run `cargo fmt` (if you installed Rust with the
+default settings using `rustup`) and it will update the code formating automatically to
+conform to the style guidelines. This is very similar manner to what `black` and running
+`tox -eblack` for Python code. For lint Qiskit uses [clippy](https://github.com/rust-lang/rust-clippy)
+which can be invoked via `cargo clippy`. For CI to pass you will need clippy to pass
+without any warnings or errors.
+
+These still get checked via the tox lint commands but if you need to update formating
+you'll need to run ``cargo fmt``.
```

Comment:
```suggestion
For formatting and lint checking Rust code, you'll need to use different tools than you would for Python. Qiskit uses [rustfmt](https://github.com/rust-lang/rustfmt) for
code formatting. You can simply run `cargo fmt` (if you installed Rust with the
default settings using `rustup`), and it will update the code formatting automatically to
conform to the style guidelines. This is very similar to running `tox -eblack` for Python code. For lint checking, Qiskit uses [clippy](https://github.com/rust-lang/rust-clippy) which can be invoked via `cargo clippy`. 

Rust lint and formatting checks are included in the the `tox -elint` command. For CI to pass you will need both checks to pass without any warnings or errors. Note that this command checks the code but won't apply any modifications, if you need to update formatting, you'll need to run `cargo fmt`.
```